src/cryptoadvance/specter/templates/includes/address-data.html

```diff
@@ -84,110 +84,91 @@ <h1>{{ _("Address details") }}</h1><br>
             let url = `{{ url_for('wallets_endpoint_api.addressinfo', wallet_alias='WALLET_ALIAS') }}`.replace("WALLET_ALIAS", this.wallet);
             var formData = new FormData();
             formData.append('address', this.address);
-            formData.append('csrf_token', '{{ csrf_token() }}');
-            try {
-                const response = await fetch(
-                    url,
-                    {
-                        method: 'POST',
-                        body: formData
-                    }
-                );
-                if(response.status != 200){
-                    showError(await response.text());
-                    return;
-                }
-                const jsonResponse = await response.json();
-                console.log(jsonResponse)
-                if (jsonResponse.success) {
-                    let descriptor = jsonResponse.descriptor;
-                    let xpubs_descriptor = jsonResponse.xpubs_descriptor;
-                    let derivation_path = jsonResponse.derivation_path;
-                    let addressIndex = jsonResponse.index;
-                    let isChange = jsonResponse.change;
-                    let walletName = jsonResponse.walletName;
-                    let address = jsonResponse.address;
-                    let walletLink = `{{ url_for('wallets_endpoint.wallet', wallet_alias='WALLET_ALIAS') }}`.replace("WALLET_ALIAS", this.wallet);
-                    let addressInfoHTML = `
-                        <qr-code style="margin: auto;" value="bitcoin:${address}" width="256"></qr-code><br>
-                        <table class="address-data-table">
-                        <tbody>
-                        <tr><td>{{ _("Address") }}:</td><td style="word-break: break-all;"><explorer-link data-type="address" data-value="${address}"></explorer-link></td></tr>
-                        <tr><td>{{ _("Label") }}:</td><td><address-label data-copy-hidden="true" data-address="${address}" data-wallet="${this.wallet}" data-label="${this.label}" data-service-id="${this.serviceId}"></address-label><br></td></tr>
-                        <tr><td>{{ _("From wallet") }}:</td><td><a href=${walletLink}>${walletName}<a></td></tr>
-                        <tr><td>{{ _("Address index") }}:</td><td>${addressIndex}</td></tr>
-                        <tr><td>{{ _("Is change address") }}:</td><td>${isChange ? '{{ _("Yes") }}' : '{{ _("No") }}'}</td></tr>
-                        <tr><td>{{ _("Used") }}:</td><td>${this.used ? 'Yes' : 'No'}</td></tr>
-                        <tr><td>{{ _("UTXO count") }}:</td><td>${this.utxo}</td></tr>
-                        <tr><td>{{ _("Amount") }}:</td><td>${this.amount} <span class="amount-price note ${this.amountPrice ? '' : 'hidden'}">${this.amountPrice}</span></td></tr>
-                    `;
+            jsonResponse = await send_request(url, 'POST', "{{ csrf_token() }}", formData);
```

Comment:
```suggestion
            let jsonResponse = await send_request(url, 'POST', "{{ csrf_token() }}", formData);
```

Reply:
Also, are those quotation marks correct or should it be back ticks:
`` `{{ csrf_token() }}` ``
Similar to sth. like this:

let url = `` `{{ url_for('auth_endpoint.toggle_hide_sensitive_info') }}` ``

Reply:
Fixed in https://github.com/cryptoadvance/specter-desktop/pull/2014/commits/f354b0b10e1e4c817dce3e044f9e3c5a815f0dc3